src/coreclr/jit/gentree.h

```diff
@@ -4848,6 +4848,11 @@ class CallArg
         return m_lateNode == nullptr ? m_earlyNode : m_lateNode;
     }
 
+    GenTree*& GetNodeRef()
```

Comment:
If you rename this to `NodeRef`, it will avoid a conflict with https://github.com/dotnet/runtime/pull/109224/files.

Reply:
Thanks, done.